src/Element.js

```diff
@@ -527,6 +527,9 @@ export default class Element {
    * @return {XML|string|*|void}
    */
   interpolate(string, data) {
+    if (typeof string !== 'function') {
+      string = FormioUtils.translateHTMLTemplate(String(string), (value) => this.t(value));
```

Comment:
I am concerned about performance here. Can we only apply this HTML template if this is the "content" component since that is really the only time it is needed?

Reply:
fixed

Reply:
what does this do?

```
(this.component.templates && Object.values(this.component.templates).some(template => !!template))
```

is this applying this for EditGrid?  Do we really need this?  Since they can override these templates, can we just add the translations using ```{{ ctx.t('Text') }}```